### PR TITLE
Edit Listing merge with Create Listing

### LINKED ISSUE #238: Add picture upload and calendar to edit listing
Currently we cannot edit pictures and availability in EditListing. For pictures, we can reuse the deletion UI that we currently have for CreateListing, and we should also add an option to upload new photos. Editing the calendar can use the same interface as CreateListing.

### LINKED ISSUE #455: Style "Choose Files" button on CreateListing page
image

We should update this button to follow our styling throughout the CreateListing page.

### LINKED ISSUE #476: Default Profile Picture
# Background

There is no default profile picture, and instead just shows up like this: 
image

# Task

* Add a profile picture by default when user doesn't have one.  Something like this should work: 
image

### PR DESCRIPTION
**Pull request by lullofthesea**
Closes #476 
Closes #238
Closes #455 

1. Edit Listing component has been merged with create listing component.  
2. Added ability to delete and add images when editing an existing listing.  
3. Added a default picture for profile and changes for other components for consistency.  
4. Added styling for select files.

Tested only with basic navigation / editing all fields.

### CODE REVIEW

**Review by teekenzie — APPROVED** [review]

**Review by aidenszeto — APPROVED** [review]
Functionality looks great! Definitely think reusing the createListing UI for this was the right move.

Here are a few things to consider: 
 - Listing calendar does not grey out unavailable dates (we can fix this in [#150](https://github.com/staynomad/Nomad-Back/issues/150)) 
 - Confirmation page after editing says "Your listing is being submitted". We could change this to "Your listing is being edited".

I'm going to push one bug fix with the user profile not getting rendered but other than that this looks good to go for a merge once the backend is approved!

**Pull request merged**
